docs/src/userguide/citation.rst

```diff
@@ -5,53 +5,15 @@ Citing Iris
 ===========
 
 If Iris played an important part in your research then please add us to your
-reference list by using one of the recommendations below.
+reference list by using the recommendations below.
 
-************
-BibTeX Entry
-************
+IRIS can be cited directly from the `GitHub repository. <https://github.com/SciTools/iris>`_
 
-For example::
+The IRIS citation does not contain the version of the software used. We
+recommend that you use the version number of the release you used, and the
+commit hash of the version you used. This will allow others to reproduce the
```

Comment:
I would only encourage a user to find the commit if they have checked out something other than a release. A release is specifically associated with a single commit, so including the release AND commit is not adding information, plus it's a harder piece of information to find.